src/components/FilterBar.jsx

```diff
@@ -76,14 +60,13 @@ export default function FilterBar(props) {
           items={sortingOptions}
           defaultValue="deadlineSoon"
```

Comment:
Look at the sort dropdown here:
https://s4us-pr-379.onrender.com/scholarships?sortDir=asc&sortField=amount.min

It's value isn't being reflected. Looks like this needs to be set based on what's currently in the query param.

Reply:
How would I do that?

Reply:
Take in the sorting option and set this. Also `value` may be a better name since it's no longer the default value.

Reply:
I think once you update your branch the value should be available in the `queryParams` prop as `sort` something. You might need to translate it to `deadlineSoon` or whatever the corresponding value is here.